OpenOversight/app/config.py

```diff
@@ -62,6 +72,12 @@ class TestingConfig(BaseConfig):
     NUM_OFFICERS = 120
     APPROVE_REGISTRATIONS = False
     SITEMAP_URL_SCHEME = 'http'
+    TWITTER_CONSUMER_KEY = 'TWITTER_CONSUMER_KEY'
+    TWITTER_CONSUMER_SECRET = 'TWITTER_CONSUMER_SECRET'
+    TWITTER_ACCESS_TOKEN = 'TWITTER_ACCESS_TOKEN'
+    TWITTER_ACCESS_TOKEN_SECRET = 'TWITTER_ACCESS_TOKEN_SECRET'
+    TWITTER_WEBHOOK_ENV = 'testing'
+    SERVER_NAME = 'localhost:5000'
```

Comment:
`SERVER_NAME` is needed to generate the `url_for(..., _external=True)` links for the twitter bot responses